packages/web/src/app/components/canvas/Canvas.tsx

```diff
@@ -114,13 +117,26 @@ const Canvas = () => {
     }
   }, [wrapperRef, router.query]);
 
+  const handleTileClick = (x: number, y: number) => {
+    console.log(document.body.clientWidth)
+    if (document.body.clientWidth <= MOBILE_WIDTH_CUTOFF) {
```

Comment:
not sure what the best way to verify that we're on mobile might be. is checking for width good enough?

Reply:
Yeah this was a quick naive check to get a POC out. Overall this entire PR needs to be re-done to be less invasive on the existing canvas code.